crates/ruff/src/rules/pylint/rules/sys_exit_alias.rs

```diff
@@ -76,15 +59,6 @@ fn get_member_import_name_alias(checker: &Checker, module: &str, member: &str) -
                     }
                 }
                 // e.g. module=os.path object=join
-                // `from os.path import *` -> `join`
-                BindingKind::StarImportation(StarImportation { module: name, .. }) => {
```

Comment:
Removing the special handling trades a false-negative to a false-positive. This is, in my view, the worse trade. Can we exit early if we see any star import? This has the advantage that we avoid the false-negative without introducing new false-positives.

Reply:
Yeah fair enough. The existing behavior is "better" and we can retain it to avoiding adding false positives here. My feeling was that it's somewhat random because so much of Ruff doesn't work if you use star imports.

For example, if you do:

```py
from sys import *

exit(1)
```

We do currently (prior to this PR) avoid raising `sys-exit-alias`, but in other rules, we consider `exit` to be the builtin `exit` and not `sys.exit` (i.e., it's special-cased here).

If you do:

```py
from sys import *

path
```

You get an undefined reference error, even though `sys.path` is a real export, since we don't know what those exports are.

If you do:

```py
from sys import *
from os import *

exit(1)
```

Then suddenly we flag `sys-exit-alias` again because of limitations in the star-import model.

So, I was left feeling like we need to redo the star handling more holistically, and that this felt like a strange thing to special-case given how much else is likely going wrong. But, it's not hard to retain this.

Reply:
I removed this handling here (not necessary), but left the special-casing in this file.